mb-solr/src/test/java/org/musicbrainz/search/solrwriter/AbstractMBWriterRecordingTest.java

```diff
@@ -9,7 +9,6 @@
 		return new ArrayList<>(Arrays.asList(new String[]{
 				"mbid", uuid,
 				"recording", "Roots and Beginnings",
-				"name", "Roots and Beginnings",
```

Comment:
I do not know what is the purpose of name here but removing it fixes the test.

Reply:
It was most likely a search field `name` redundant with the search field `recording` of [Recording](https://musicbrainz.org/doc/MusicBrainz_API/Search#Recording).